banzai_nres/tests/test_e2e.py

```diff
@@ -283,6 +283,17 @@ def test_if_stacked_arc_frame_was_created(self):
         run_check_if_stacked_calibrations_have_extensions('double', ['WAVELENGTH', 'FEATURES'])
         run_check_if_stacked_calibrations_are_in_db('*a00.fits*', 'DOUBLE')
 
+    def test_quality_of_wavelength_calibration(self, calibration_type='double', primaryextension=1):
+        created_stacked_calibrations = []
+        for day_obs in DAYS_OBS:
+            created_stacked_calibrations += glob(os.path.join(DATA_ROOT, day_obs, 'processed',
+                                                              '*' + calibration_type.lower() + '*.fits*'))
+        for cal in created_stacked_calibrations:
+            hdulist = fits.open(cal)
+            quality_metrics = hdulist[primaryextension].header
+            assert quality_metrics['PRECISN'] < 30
```

Comment:
This will need to be changed once the branch is rebased because this header keyword has been renamed to RVPRECSN.